languages/java8/template/src/main/java/com/algorithmia/algorithm/Algorithm.java

```diff
@@ -0,0 +1,54 @@
+package com.algorithmia.algorithm;
+
+import com.algorithmia.development.*;
+import com.algorithmia.*;
+
+class Algorithm {
+    public String apply(String input) throws RuntimeException {
+        return "hello ".concat(input);
+    }
+
+    public static void main(String[] args) {
+        Algorithm defs = new Algorithm();
```

Comment:
why is it called `defs`?

Reply:
Can be renamed, we can also pass `this` if we wanted

Reply:
Does `defs` refer to definitions?

Reply:
Does `defs` stand for definitions?